code/data_structures/src/DoubleLinkedList/Linkedlist.cpp

```diff
@@ -0,0 +1,171 @@
+#include<bits/stdc++.h>
+using namespace std;
+#define endl "\n"
+
+class node{  //node of linked list
+     public:
+     int data;
+     node* next;
```

Comment:
This is not a doubly linked list.

(based on the directory you have placed the code under)